webapi/Extensions/SemanticKernelExtensions.cs

```diff
@@ -165,12 +220,45 @@ private static Task RegisterSkillsAsync(IServiceProvider sp, IKernel kernel)
                 }
                 catch (SKException ex)
                 {
-                    var logger = kernel.LoggerFactory.CreateLogger(nameof(Kernel));
                     logger.LogError("Could not load skill from {Directory}: {Message}", subDir, ex.Message);
                 }
             }
         }
 
+        // Native skills
+        if (!string.IsNullOrWhiteSpace(options.NativeSkillsDirectory))
```

Comment:
Is this functionality documented in the readme?

Reply:
Yeah I added some hearty sections to the README about Adding Custom Skills and the Custom Setup Hooks, lmk if you think there's other details needed

Reply:
Ah I missed the first part, thanks!